core/variant/typed_array.h

```diff
@@ -59,6 +59,20 @@ class TypedArray : public Array {
 	_FORCE_INLINE_ TypedArray() {
 		set_typed(Variant::OBJECT, T::get_class_static(), Variant());
 	}
+	_FORCE_INLINE_ TypedArray(const Vector<std::add_pointer_t<T>> &p_vector) {
+		resize(p_vector.size());
+		int offset = 0;
+		for (const std::add_pointer_t<T> &element : p_vector) {
+			insert(offset++, element);
+		}
+	}
+	_FORCE_INLINE_ TypedArray(const List<std::add_pointer_t<T>> &p_list) {
+		resize(p_list.size());
+		int offset = 0;
+		for (const std::add_pointer_t<T> &element : p_list) {
+			insert(offset++, element);
```

Comment:
Why `insert`? If you resize first, then insert elements, you should get twice the elements, no? I feel like I'm missing something here 😅

Reply:
I might've brainfarted here, oops